beacon-chain/chaintest/backend/simulated_backend_test.go

```diff
@@ -24,7 +24,7 @@ func TestGenerateBlocks(t *testing.T) {
 		t.Fatalf("Could not create a new simulated backedn %v", err)
 	}
 
-	initialDeposits, err := generateInitialSimulatedDeposits(1000)
```

Comment:
No need to run tests for thousands of validators, instead we should aim to keep these CI tests as small as possible.